content/blog/open-sourcing-workshops-on-demand-part-5-create-a-workshop.md

```diff
@@ -0,0 +1,226 @@
+* **R﻿eset and ldap** entries are to be used by backend server automation if dedicated reset scripts and ldap authentication are required by the workshop.
+* **A﻿ session type:** Workshops-on-Demand by default
+* **A﻿ location:** If your setup includes multiple production sites, use this field to allocate workshops according to your needs. In the case of the HPE Developer Community, some workshops can only run on a HPE GreenLake cloud environment. As a consequence, the location is set to greenlake in this case.
+* **A﻿vatar, role and replayLink** are superseeded by entries in the replay table. I will explain later.
+
+![](/img/wod-db-go-2.png "Workshop's fields in the Database #2.")
+
+* **Compile:** This entry will be filled with the name of a script to be compiled at deployment time. This feature allows for instance the admin to hide login scripts and credentials in non-editable executable files.
+* **Varpass:**  This defines whethere a workshop require some password variable to be leveraged or not.
+* **R﻿eplayId:** This entry links the dedicated replay video to the workshop. it enables the presence of the replay in the learn more page of the workshop.
+* **W﻿orkshopImg:** As part of the lifecycle of the workshop, several emails are sent to the student. A workshop image is embbeded in the first emails.
+* **B﻿adgeImg:** As part of the lifecycle of the workshop, several emails are sent to the student. In the final email, a badge is included. It allows the student to share its accomplishment on SoME like linkedin for instance.
+
+***N﻿ote:*** B﻿oth W﻿orkshopImg and B﻿adgeImg are located on the same remote web server.
+
+* **B﻿eta:** Not implemented yet :-)
+* **Category:** The workshops' registration portal proposes several filters to display the catlog's content. You can view all workshops, the most poular ones, or by category. Use this field to sort workshops accordingly.
+* **A﻿lternateLocation:** Not implemented yet. The purpose is allow automation of the relocation of a workshop in case of primary location's failure.
+* **D﻿uration:** All workshops are time bombed. You will define here the time alloacted to perform the workshop.
+* **A﻿ctive:** Tag to set to enable visibility of the workshop's tile in the registration portal.
+
+I﻿f you feel you need more details about the registration process, please take a look at the **Register Phase** paragraph in [the following introductionary blog](https://developer.hpe.com/blog/willing-to-build-up-your-own-workshops-on-demand-infrastructure/).
+
+I﻿n the **Replays table:**
+
+A﻿ new entry will need the following:
+
+![](/img/wod-db-go-3.png)
+
+* **Id:** This entry will be filled with the name of a script to be compiled at deployment time. This feature allows for instance the admin to hide login scripts and credentials in non-editable executable files.
+* **Avatar:**  This defines whethere a workshop require some password variable to be leveraged or not.
```

Comment:
"whether" is misspelled
"require" should be "requires"

The construction of this sentence is a little odd. Try this:
"This defines whether or not a workshop requires a password variable needs to be leveraged."